helicone-node/tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "compilerOptions": {
     "declaration": true,
-    "target": "es6",
-    "module": "commonjs",
+    "target": "es2020",
+    "module": "CommonJS",
+    "moduleResolution": "Node",
```

Comment:
what does this do? Does this match OpenAI's pacakge?

Reply:
matching OpenAI's package: https://github.com/openai/openai-node/blob/master/tsconfig.json#L4C3-L7C19